### PR TITLE
test: Calculate proper snappoints in matcher

### PR DESCRIPTION
**Pull request by lautarorodriguezmrf**

### CODE REVIEW

File: e2e/utils/snapPoints.js [hunk 1]
```diff
@@ -4,62 +4,49 @@ async function getCurrentVerticalPosition(browser, cardSelector) {
 	return await browser.executeAsync(async(cardSelectorBrowser, done) => {
 		const y = document.querySelector(cardSelectorBrowser).getBoundingClientRect().y;
 
-		done(y);
+		done({ card, positioner });
```
Comment (davidramosmrf) [inline e2e/utils/snapPoints.js]: where are you getting these values?

File: e2e/utils/snapPoints.js [hunk 2]
```diff
@@ -4,62 +4,49 @@ async function getCurrentVerticalPosition(browser, cardSelector) {
...
+	}, cardSelector);
 
-const verifyPercentageSnapPoint = async(browser, cardSelector, expectedSnapPointPercentage)=>{
-	const currentPercentage = await browser.executeAsync(async(cardSelectorBrowser, done) => {
-		const percentage = (document.querySelector(cardSelectorBrowser).getBoundingClientRect().y)/window.innerHeight;
+	/**
+	 * Make sure to keep this compensation matching the one we are doing inside flowcards code:
+	 * https://github.com/Marfeel/flowcards/blob/master/packages/experience-web/src/transitioner/useAbsoluteSnapPoints/useAbsoluteSnapPoints.tsx#L46
+	 */
```
Comment (davidramosmrf) [inline e2e/utils/snapPoints.js]:
```suggestion
	/**
	 * ⚠️ WARNING ⚠️ 
	 * Make sure to keep this compensation matching the one we are doing inside flowcards code:
	 * https://github.com/Marfeel/flowcards/blob/master/packages/experience-web/src/transitioner/useAbsoluteSnapPoints/useAbsoluteSnapPoints.tsx#L46
	 */
```

**Review by davidramosmrf — DISMISSED** [review]
LGTM

File: e2e/utils/snapPoints.js (hunk 1 above)
Comment (lautarorodriguezmrf) [reply e2e/utils/snapPoints.js]: it was a bad merge, reverted

**Review by davidmohringmrf — APPROVED** [review]

**Review by davidramosmrf — APPROVED** [review]

**Review by jonayrodriguezmrf — APPROVED** [review]

**Pull request merged**